index/scorch/scorch.go

```diff
@@ -157,29 +153,15 @@ func configForceSegmentTypeVersion(config map[string]interface{}) (string, uint3
 }
 
 func (s *Scorch) paused() uint64 {
```

Comment:
I prefer a name like `NumEventsBlocking()` as `paused` implies a particular use case that only really applies to one kind of event.

Reply:
done!